doc/foreign-code.rst

```diff
@@ -195,10 +195,10 @@ demonstrates how to access the bound C library functions and values:
 From here, one only needs to run ``dune build ./foo.exe`` to generate the
 stubs and build and link the example ``foo.exe`` program.
 
-Complete information about the ctypes combinators used above is available at
-the ctypes_ project.
+Complete information about the ``ctypes`` combinators used above is available at
```

Comment:
Since ctypes isn't an identifier that appears in source code, it should just be Ctypes.